SECURITY.md

```diff
@@ -0,0 +1,59 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+<!-- BEGIN PRAVEGA SECURITY.MD V0.0.1 BLOCK -->
+
+## Security
+
+Pravega takes the security of our software products and services seriously, which includes all source code repositories managed through our GitHub organizations.
+
+If you believe you have found a security vulnerability in any Pravega owned repository, please report it to us as described below.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report them to the Pravega Security Team via email(mailto:security@pravega.io).
+
+You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message.
+
+Please include the requested information listed below (as much as you can provide) to help us better understand the nature and scope of the possible issue:
+
+  * Product name and version containing the vulnerability
+  * Type of issue (e.g. buffer overflow, SQL injection, cross-site scripting, etc.)
+  * Full paths of source file(s) related to the manifestation of the issue
+  * The location of the affected source code (tag/branch/commit or direct URL)
+  * Any special configuration required to reproduce the issue
+  * Step-by-step instructions to reproduce the issue
+  * Proof-of-concept or exploit code (if possible)
+  * Impact of the issue, including how an attacker might exploit the issue
+
+This information will help us triage your report more quickly.
+
+
+## Preferred Languages
+
+We prefer all communications to be in English.
+
+## Policy
+
+Pravega follows the principle of [Coordinated Vulnerability Disclosure](https://www.cisa.gov/coordinated-vulnerability-disclosure-process).
```

Comment:
Have you got a confirmation about this? cc @fpj / @derekm .

Reply:
@ravisharda -- I think that's a holdover from when @ravibeta was reusing Dell's policy. Declaring alignment with the CVD process may make sense.

As for Core Infrastructure Initiative's Best Practices guidelines at the "passing" level, our policy must at least meet these criteria:
* The project's initial response time for any vulnerability MUST be less than or equal to 14 days.
* There MUST be no unpatched vulnerabilities of medium or higher severity that have been publicly known for more than 60 days. [Severity determined by Common Vulnerability Scoring System (CVSS).]
* Projects SHOULD fix all critical vulnerabilities rapidly after they are reported.
* It is SUGGESTED that at least one of the static analysis tools used for the static_analysis criterion include rules or approaches to look for common vulnerabilities in the analyzed language or environment.
* All medium and higher severity exploitable vulnerabilities discovered with static code analysis MUST be fixed in a timely way after they are confirmed.
* All medium and higher severity exploitable vulnerabilities discovered with dynamic code analysis MUST be fixed in a timely way after they are confirmed.